### ISSUE TITLE
SmartCity-Accessibility Server Side Network Tests #59 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#369]
Build 'SmartCity-Accessibility Server Side Network Tests' is failing!

Last 50 lines of build :
```
[...truncated 5.24 KB...]
 :: Spring Boot ::        (v1.5.2.RELEASE)

-- Product name: JxMaps
-- Licensed version: 1.x
-- Licensed to: 
-- License type: Evaluation
-- Generation date: Dec 7, 2016
-- Expiration date: Aug 25, 2017
-- License info: Single-user license
-- Current date: May 23, 2017
JxMaps license valid.
Running smartcity.accessibility.services.NavigationServiceTest
Tests run: 3, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 19.57 sec - in smartcity.accessibility.services.NavigationServiceTest
Running smartcity.accessibility.navigation.NavigationTest
88157718
שדרות תל חי
Tests run: 4, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 5.293 sec - in smartcity.accessibility.navigation.NavigationTest
Running smartcity.accessibility.database.ParseDatabaseTest
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 4.111 sec <<< FAILURE! - in smartcity.accessibility.database.ParseDatabaseTest
testGetHighestBy(smartcity.accessibility.database.ParseDatabaseTest)  Time elapsed: 0.266 sec  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[7EpEj8TgFO]> but was:<[ufJtidwhrF]>
	at smartcity.accessibility.database.ParseDatabaseTest.testGetHighestBy(ParseDatabaseTest.java:154)


Results :

Failed tests: 
  ParseDatabaseTest.testGetHighestBy:154 expected:<[7EpEj8TgFO]> but was:<[ufJtidwhrF]>

Tests run: 15, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:35 min
[INFO] Finished at: 2017-05-23T17:27:29+00:00
[INFO] Final Memory: 33M/203M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/sharedspace/SmartCity-Accessibility/server-side/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure
```

Changes since last successful build:
- [Alexander Kaplan] fe806f32d705a98fb2375c868377a4d655c3e657 - implemented get hight by #365

- [Alexander Kaplan] d8c10f67a12475fd30eefe684d4b01cc4ee3bc77 - added test for new parseDatabase method #365 and slight improvemnt to

[View full output](http://kapalex.ddns.net:8080/job/SmartCity-Accessibility%20Server%20Side%20Network%20Tests/59/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
